parse/mod_test.go

```diff
@@ -26,16 +25,10 @@ func TestMapModToPkg(t *testing.T) {
 				{
 					Path:    "github.com/pkg/errors",
 					Version: "v0.8.1",
-					Time:    time.Now(),
```

Comment:
Just a question here: given we are removing these fields from the tests, does this also mean the actual output of the dep report is also losing this information?

Reply:
That's a good question. No it does not. These fields were not being used by the report generator. Instead they gathered the info from the source's API. Because these fields are not available to us when we parse go.mod directly, I decided to remove them from the type definition.

Reply:
Makes sense! Thanks for considering the question.